### PR TITLE
Public nonce key

### PR DESCRIPTION
**Pull request by LiorAgnin**
subj

### CODE REVIEW

File: src/sdk.ts [hunk 1]
```diff
@@ -105,15 +105,9 @@ export class FuseSDK {
     const initialFees = BigInt(txOptions.feePerGas);
     this.setWalletFees(initialFees);
 
-    if(isIndependentTransaction) {
+    if (isIndependentTransaction) {
```
Comment (leonprou) [inline src/sdk.ts]: not sure I like this name because independentTransaction pretty vague.
maybe we can call this `newNonceSequence`

Comment (leonprou) [reply src/sdk.ts]: @MusabShakeel576 cc

Comment (LiorAgnin) [reply src/sdk.ts]: I concur. A more suitable name would be `useNonceSequence`. 
WDYT @leonprou ?

**Review by MusabShakeel576 — APPROVED** [review]
LGTM

File: src/sdk.ts (hunk 1 above)
Comment (leonprou) [reply src/sdk.ts]: Sounds great!

**Pull request merged**